addons/backgrounds/src/models/BackgroundSelectorItem.ts

```diff
@@ -0,0 +1,7 @@
+export interface BackgroundSelectorItem {
+  id: string;
+  title: string;
+  onClick: () => void;
+  value: string;
+  right?: any;
```

Comment:
Your changes at https://github.com/storybooks/storybook/pull/5535/files#diff-f01e7bd996316ee937af51de4ab40f37R30 are not quite the same as the original. The original would be `right? any`, while yours is actually `right: any` since you're always setting the `right` property.


Also: Do we not have config in place to type this as a React/JSX element?